docs/installing.rst

```diff
@@ -27,21 +28,54 @@ In your ``conf.py`` file:
     :ref:`supported-dependencies`
         Officially Supported versions of Python, Sphinx, and other dependencies.
 
+
 .. note::
 
    Adding this theme as an extension is what enables localization of theme
    strings in your translated output. If these strings are not translated in
    your output, either we lack the localized strings for your locale, or you
    are using an old version of the theme.
 
+   ..
+      comment about this note: it's possibly not necessary to add the theme as an extension.
+      Rather, this is an issue caused by setting html_theme_path.
+      See: https://github.com/readthedocs/readthedocs.org/pull/9654
+
+
+.. _howto_upgrade:
+
+How to upgrade
+--------------
+
+.. tip:: 
+    **Read the Docs users:** To know which version of ``sphinx-rtd-theme`` you are running, you can always check your latest build outputs. One or more steps in the documentation build will install python packages, one of which is ``sphinx-rtd-theme``.
+
+Adding ``sphinx-rtd-theme`` to your project's dependencies will make pip install the latest compatible version of the theme.
+
+If you want to test a **pre-release**, you need to be explicit about the version you specify.
+Otherwise, pip will ignore pre-releases. Add for instance ``sphinx-rtd-theme==1.1.0b3`` to test a pre-release.
+
+.. tip::
+    We always recommend that you pin the exact version of Sphinx that your project is built with.
+    We'll make sure not to release sphinx-rtd-theme that installs together with a Sphinx theme that it is incompatible with.
```

Comment:
True. It was wrongly phrased and I find it a bit tricky. Is the meaning better here?

```suggestion
    We won't release sphinx-rtd-theme without marking its compatibility with Sphinx. So if you do not pin `sphinx-rtd-theme` itself, you will always get the *latest compatible* release.
```

Reply:
That is much clearer :+1: